charts/spire/charts/spire-agent/values.yaml

```diff
@@ -206,8 +206,14 @@ workloadAttestors:
   k8s:
     ## @param workloadAttestors.k8s.enabled Enables the Kubernetes workload attestor
     enabled: true
-    ## @param workloadAttestors.k8s.skipKubeletVerification If true, kubelet certificate verification is skipped
-    skipKubeletVerification: true
+    verification:
+      ## @param workloadAttestors.k8s.verification.type What kind of verification to do against kubelet. Valid options are [hostCert, apiServerCA, skip]
+      type: hostCert
+      hostCert:
+        ## @param workloadAttestors.k8s.verification.hostCert.basePath Path where kubelet places its certificates
+        basePath: /var/lib/kubelet/pki
```

Comment:
Would be great if we can pull this off. Im worried there will be a lot of variance where this base path is depending on the platform. Whats your experience been with that?

Reply:
So far, not a lot. I'm sure some distros will use a different value. But we have the same issue with the csi driver too.